docs/network/evm-nodes.md

```diff
@@ -40,16 +37,39 @@ The release page provides static binaries for Linux systems (for amd64 and arm64
 As an alternative, you can use the minimal Docker image [tezos/tezos-bare](https://hub.docker.com/r/tezos/tezos-bare/tags?name=octez-evm-node) with a tag that includes `octez-evm-node`.
 These images contain the correct version of the binary.
 
-## Initializing the data directory
+## Configuring the node
+
+The node uses the following main parameters.
+In versions of the `octez-evm-node` binary prior to version 0.20, you must specify the parameters as arguments to most commands.
+In version 0.20 and later, you can set environment variables instead of using the arguments.
+
+- The directory where it stores its data, which you specify in the `--data-dir` argument or in the `EVM_NODE_DATA_DIR` environment variable.
+The default directory is `$HOME/.octez-evm-node`.
+The following instructions use the variable `<EVM_DATA_DIR>` to represent this directory.
+- The location of its configuration file, which you specify in the `--config-file` argument or in the `EVM_NODE_CONFIG_FILE` environment variable.
+The default location is the file `config.json` in the data directory.
```

Comment:
I would explain first that this is needed only for people that want to separate the configuration from the application data (which is the recommended approach in kubernetes for instance), this way people that are not interested can quickly skim over to the next sentence.

Reply:
Added this:

> You can set a different location of the file if you want to separate the data directory from the node configuration, such as often happens in Kubernetes.

Reply:
Maybe it can be before the previous sentence? I feel it would be clearer but feel free to resolve.

Reply:
K, moved it up a sentence.